### PR TITLE
COR-4230 & COR-4261: Ensure focus & meaningful title to iframe for WCAG 2.1

### PR DESCRIPTION
**Pull request by jenn-jenn**
## What did you implement
- add `title` attribute to `iframe`
- focus to iframe after launching with `focus()` method 
- ~~`aria-hidden` to `app` and `footer` (if any) to hide from accessibility API when Connect is launched~~ moved logic out of library
- bump to 2.2.1-beta.4

## How can we verify it
- integrate Connect following this guide [https://reference.humanapi.co/docs/web-guide](https://reference.humanapi.co/docs/web-guide)
-- Include [https://cdn.humanapi.co/humanapi-connect-client@2.2.1-beta.4.js](https://cdn.humanapi.co/humanapi-connect-client@2.2.1-beta.4.js) in your code OR `npm i humanapi-connect-client@2.2.1-beta.4` in your project
- after launching Connect, focus should be on `iframe`
-- If using Google Chrome, you can verify using the `Create Live Expression` (the eyeball icon) in DevTools and adding `document.activeElement` in the console
-- pressing tab will go to the next item in the iframe

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -170,6 +171,11 @@ function load(options) {
         logger("Cleaning up elements");
         document.getElementById("human-api").remove();
         document.getElementById("human-api-connect-modal-overlay").remove();
+        document.getElementById("app").setAttribute('aria-hidden', 'false');
+        const footer = document.getElementById("footer");
```
Comment (maumont) [inline lib/index.js:175]: so here we assume the consumer of connect has an element called `app` and perhaps a `footer` one right? 
I think that applies to our way of using connect in our other apps but might not apply on other client pages

**Review by rafasalazarzx — APPROVED** [review]
👍

File: lib/index.js (hunk 1 above)
Comment (jenn-jenn) [reply lib/index.js:175]: As discussed, I will set `aria-hidden` for the element right after body, taking into account that not all apps might have this `app` id

File: lib/index.js [hunk 2]
```diff
@@ -184,7 +185,7 @@ function load(options) {
         window.removeEventListener("message", onMessage, false);
         window.removeEventListener("click", mountOnClickListener);
     }
-
+6
```
Comment (maumont) [inline lib/index.js:188]: Random `6` here 🤔

Comment (jenn-jenn) [reply lib/index.js]: fixed!

**Review by maumont — APPROVED** [review]

**Comment by maumont** [comment]
@jenn-jenn need to double check, but I think we have to remove the `beta` files inside `/build`

**Pull request merged**